tox.ini

```diff
@@ -34,8 +34,8 @@ deps =
 # minimum numpy to 1.15. mpl while not a dependency, it's required for the
 # tests, and would pull up a newer numpy version if not pinned.
 
-    oldestdeps: astropy==3.1.2
-    oldestdeps: numpy==1.15
+    oldestdeps: astropy==4.0.6
```

Comment:
I like to keep this at 4.0 if possible, as that is what actually is in the configs.

Reply:
Makes sense. Done.

Reply:
Hmm, unpinning bugfix broke the job:

```
E   astropy.utils.exceptions.AstropyWarning: leap-second auto-update failed due to the following exception: IERSStaleWarning('leap-second file is expired.')
```

Reply:
So how should I proceed with this? Please advise. Thanks!

Reply:
I think I am going to add back the minversion pin for now, and make it consistent everywhere.

Reply:
hmm, but that begs the question of whether we're e.g. 4.1 compatible as chronologically 4.0.6 came out very recently. (It's a fundamental issue with LTS bugfix versions, not just here)

Reply:
I dunno. The error came from astropy-helpers that was tied to 4.0.1. Since we don't really update astropy-helpers anymore, the only option is to update the bugfix version of LTS.

Reply:
I'm pretty sure the helpers is a red herring, at least I don't see anything related to the in the tracelog here, but all purely astropy core. Am I looking at the wrong place: https://github.com/astropy/astroquery/runs/3750915235#step:5:136

Reply:
Oh, you are right! I was thinking about the coverage error with astropy 3.x. My bad!

`E   astropy.utils.exceptions.AstropyWarning: leap-second auto-update failed due to the following exception: IERSStaleWarning('leap-second file is expired.')` means the leap second data file is too old... I also dunno how to fix except to bump the bug fix version...

Reply:
Unless... we can add that warning to be ignored in `setup.cfg`?

Reply:
yes, I would say that's an astropy issue that shouldn't really matter for astroquery (as it surely comes up for those with that old astropy anyway whatever they do), so it's OK if we ignore it for now.

Reply:
OK done 🤞